packages/headless/src/chat.ts

```diff
@@ -70,7 +85,14 @@ enum ActionType {
   PostCallError,
   AddPendingMessage,
   ActivateMessage,
+  /**
+   * for when a send message fails
```

Comment:
Let's capitalize the first letter in the description. Let's stick to this convention. 

```suggestion
   * For when a send message fails
```